projects/rero/ng-core/src/lib/menu/menu-item.ts

```diff
@@ -571,49 +599,37 @@ export class MenuItem implements MenuItemInterface {
   }
 
   /**
-   * Count
-   * @return integer, number of children
+   * Is the menuItem is enabled
+   * @return boolean
    */
-  count(): number {
-    return this._children.length;
+  enabled(): boolean {
```

Comment:
Why not make `isEnabled` and delete `enable` and `disable` function ?

Reply:
I change `enabled()` to `isEnabled()` to be consistant with others interface functions.
I will keep the `enable()` and `disable()` functions as setter for the private `_enabled` attribute.